storagemarket/deal_execution.go

```diff
@@ -194,6 +192,20 @@ func (p *Provider) execDealUptoAddPiece(ctx context.Context, pub event.Emitter,
 		p.dealLogger.Infow(deal.DealUuid, "deal has already been handed over to the sealing subsystem")
 	}
 
+	// as deal has already been handed to the sealer, we can remove the inbound file and reclaim the tagged space
```

Comment:
Is the inbound file actually removed here? If not we should change this comment.

Reply:
good catch, removing file now.